src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -88,6 +91,35 @@ const ActiveState = ({
 
                 switch (item.type) {
                     case 'comment':
+                        if (hasReplies) {
```

Comment:
I'm wondering if it would be cleaner if we deferred the `hasReplies` check in the `ActivityThread` component